app/controllers/api/api_controller.rb

```diff
@@ -455,9 +458,8 @@ def pept2interpro_helper
         output.map do |_k, v|
           v[:ipr].each do |value|
             ipr_entry = ipr_mapping[value[:code]]
-
-            value[:name] = ipr_entry.name
-            value[:type] = ipr_entry.category
+            value[:name] = "" if ipr_entry.nil? else ipr_entry.name 
```

Comment:
this is python style ;) In ruby `= test ? value1 : value2` is more common

Reply:
You're right :) I'll change it!